newclient/src/modules/blocks/handlers/BlockStats/BlockStats.vue

```diff
@@ -84,7 +84,7 @@ export default class BlockStats extends Mixins(NumberFormatMixin) {
         gh: 'Gh/s',
         mh: 'Mh/s'
     }
-    initialLoadedBlock!: number
+    initialLoadedBlock: number = 0
```

Comment:
I had to change the way this was getting declared because the computed property (getLatestBlock) wasn't getting updated with the new values for some reason. I couldn't find too much information why it wouldn't let me keep the ```!``` but it was saying ```A definite assignment assertion '!' is not permitted in this context.```